main/olive/plugins.yml

```diff
@@ -128,6 +128,16 @@
   description: |
     Manage your Tutor-powered Open edX installation from the browser.
 
+- name: wordpress
+  src: tutor-contrib-wordpress>=1.1.0,<2.0.0
+  url: 'https://github.com/CodeWithEmad/tutor-contrib-wordpress'
+  author: Emad Rad <codewithemad@gmail.com>
+  maintainer: Emad Rad <codewithemad@gmail.com>
+  description: |
+    Tutor plugin for WordPress. This plugin is designed to assist with the
```

Comment:
Try not to start your plugin description with "Tutor plugin...". All items in this list are tutor plugins :) Also, keep in mind that the first 128 characters of the first line of the description are used as "short description": https://github.com/overhangio/tutor/blob/1f43ac5f87701dd9192c00f0e939ba3be3c14c5a/tutor/plugins/indexes.py#L62
This will be useful for displaying tutor plugins in `tutor search ...` commands.